community/releases/release-manager-runbook/index.md

```diff
@@ -0,0 +1,329 @@
+
+    a. Click **New Item** on the left hand side of the main menu.
+
+    b. Set **Name** to Flow_BuildAndValidate_${RELEASE} (e.g., `1_18_x`). Note
+    that the newer versions of Jenkins seem to disallow the `.` character in the
+    job name, so will want to `s/./_`.
+    
+    c. Set **Copy from** to Flow_BuildAndValidate.
+
+    d. In the **Build Triggers** section, paste cut text from the second-oldest
+    flow.
+    
+    e. Set **GITHUB_REPO BRANCH** to `${RELEASE_BRANCH}`.
+    
+    f. Set **PROCESS_GITHUB_REPO_BRANCH** TO `${RELEASE_BRANCH}`.
+    
+7. At this point, there should exist the following Flow_BuildAndValidate_* jobs:
+
+    a. Flow_BuildAndValidate_${RELEASE-3} (DEACTIVATED)
+    
+    b. Flow_BuildAndValidate_${RELEASE-2} (BUILDING NIGHTLY)
+    
+    c. Flow_BuildAndValidate_${RELEASE-1} (BUILDING NIGHTLY)
+    
+    d. Flow_BuildAndValidate_${RELEASE} (BUILDING NIGHTLY)
+    
+    e. Flow_BuildAndValidate (master, BUILDING NIGHTLY)
+    
+8. Run the Flow_BuildAndValidate_${RELEASE} job.
+
+    a. Select `stable` for **HALYARD_RELEASE_TRACK**.
+    
+    b. This will automatically update the [changelog gist](https://gist.github.com/spinnaker-release/4f8cd09490870ae9ebf78be3be1763ee)
+    on GitHub.
+    
+9. Add the new Flow_BuildAndValidate_${RELEASE} job to the public
+[Build Statuses page](https://www.spinnaker.io/community/contributing/build-statuses/#nightly-and-release-integration-tests).
+Ping #dev with some version of this message, including a link to the correct
+section of the changelog gist:
+
+> The release window for Spinnaker $VERSION is now open!  This means that
+> release branches have been cut from master and those branches are only
+> accepting fixes for existing features.  Please contact $YOUR_NAME
+> (slack: $YOUR_SLACK_ID, github: $YOUR_GITHUB_ID, or email: $YOUR_EMAIL) if you
+> would like a fix cherry-picked into the release or you would like to highlight
+> a specific fix or feature in the release’s changelog. If you’d like to jog
+> your memory of everything to be released with Spinnaker $VERSION, see the raw
+> changelog here: $LINK_TO_CHANGELOG.
+
+10. Share a curated changelog with any partners from the community who wants to
+add notes.
+
+11. When the Flow_BuildAndValidate_${RELEASE} job passes, ping #dev with a
+message that release candidate is now validated and can be tested by running:
+
+```
+hal config version edit --version ${RELEASE_BRANCH}-latest-validated
+```
+
+## Second Monday of the release window
+
+1. Check for any PRs waiting to be [cherry-picked](https://github.com/search?q=org%3Aspinnaker+label%3Acherry-pick+state%3Aopen+type%3Apr).
+(You can further restrict the query by adding a constraint like +base:release-1.18.x to the URL.)
+
+2. Rerun the Flow_BuildAndValidate_${RELEASE} job and get a green build.
+
+3. Create a new gist for this release.
+
+    a. Log into GitHub as spinnaker-release and create a new gist to hold the
```

Comment:
Currently, I think only the Google team has access to log into GitHub as spinnaker-release. I'm happy to do these steps for Ethan this time around, but should we consider giving access to all Spinnaker release managers?

Reply:
Yes, I think it makes sense for everyone to have the same access (even this time).

Reply:
You may want to consider transferring the doc with the 2FA image to the `spinnaker.io` domain, maybe owned by admin@spinnaker.io

Reply:
Well, the spinnaker-release password lives in the internal password tool, so I guess we'd need to share that too...going to defer to @plumpy on whether sharing that password is okay?

Reply:
Let's continue the conversation on how best to handle spinnaker-release permissions in #build-process; going to merge this PR as-is for now but of course happy to update if needed when we land on a solution here.